src/modules/getBankNameFromCardNumber/index.ts

```diff
@@ -15,7 +15,7 @@ export interface IBank {
 function getBankNameFromCardNumber(digits?: number | string): string | null | undefined {
 	if (!digits) return;
 
-	if (digits && digits.toString().length === 16) {
+	if (digits && digits.toString().length >= 6 && digits.toString().length <= 16) {
```

Comment:
`digits.toString().length`called twice.

Reply:
Do you think it's okay to define a const for the digits length?

Reply:
Exactly